src/SlashCommands.js

```diff
@@ -282,6 +282,63 @@ var commands = {
             }
         }
         return reject(this.getUsage());
+    }),
+
+    // Verify a user, device, and pubkey tuple
+    verify: new Command("verify", "<userId> <deviceId> <deviceSigningKey>", function(room_id, args) {
+        if (args) {
+            var matches = args.match(/^(\S+) +(\S+) +(\S+)?$/);
```

Comment:
FWIW the other code in this file follows this Regex approach to parse out inputs. Probably should be properly validated and sanitized, but that's kinda unrelated to this change.